streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -52,10 +53,8 @@ public void before() {
     }
 
     @After
-    public void cleanup() {
-        if (stateDir.exists()) {
-            Utils.delete(stateDir);
-        }
+    public void cleanup() throws IOException {
+        Utils.delete(stateDir);
```

Comment:
Have you verified that the `exists` was not there for a reason? Otherwise we may get failures during cleanup, which may obscure the real issue?